README.md

```diff
@@ -29,6 +27,10 @@ An example build may look like:
 	  sudo ln -s /usr/lib/dovecot/lib21_fts_elastic_plugin.so /usr/lib/dovecot/modules/lib21_fts_elastic_plugin.so
 
 ## Configuration
+Create an index in Elasticsearch:
+
+	curl -X PUT "localhost:9200/m?pretty"
```

Comment:
It is highly recommended to use prepared elastic schema as is written later in readme:
curl -X PUT "http://elasticIP:9200/m?pretty" -H 'Content-Type: application/json' -d "@elastic7-schema.json"